lamindb/curators/core.py

```diff
@@ -427,7 +490,9 @@ def __init__(
                 # in almost no case, an index should have a pandas.CategoricalDtype in a DataFrame
                 # so, we're typing it as `str` here
                 index = pandera.Index(
-                    schema.index.dtype if not feature.dtype.startswith("cat") else str
```

Comment:
Oh, this was the bug, I guess. And there was no test for an index with a categorical dtype. Got it.